osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
> but keep it up to the model to really check if this requirement was fulfilled.

Either it has to be fulfilled by the tool or not. Shouldn't be too complicated for us, I just wanted to clarify.

**Reference frame:**
At the moment I cannot really follow the discussion. Usually you have a position (world, relative, road, whatever), an orientation and a contour (or bounding box) _relative_ to the given position and orientation.

E.g.: 
Object position (reference point): x=10, y=20, z=0
Object orientation: hdg=30°, pit=0°,rol=0°
Object contour: p1[2;0], p2[-2;-2];p3[-2;2]

This results in a rectangle at x=10, y=20, rotated by 30°:
![example](https://cloud.githubusercontent.com/assets/25342566/23017519/e1ce5a06-f43a-11e6-96c3-1af84a968832.jpg)